packages/usehooks-ts/src/useDebounceCallback/useDebounceCallback.ts

```diff
@@ -77,39 +77,52 @@ export function useDebounceCallback<T extends (...args: any) => ReturnType<T>>(
   options?: DebounceOptions,
 ): DebouncedState<T> {
   const debouncedFunc = useRef<ReturnType<typeof debounce>>()
+  const isPendingRef = useRef(false)
 
+  const parsedFunc = useCallback(
+    (...args: Parameters<T>) => {
+      const result = func(...args)
+      isPendingRef.current = false
```

Comment:
Would it make sense to put this line into a `finally` block, so that it even works as expected if `func` throws an exception?

Reply:
wow, that's coolll, fixed!